backend/e4h-services/ingestion-service/app/utils/amc_scheduler_service_client.py

```
@@ -0,0 +1,88 @@
+        headers = {
+            "Content-Type": "application/json"
+        }
+        
+        payload = {
+            "RequestInfo": request_info.model_dump(by_alias=True, exclude_none=True),
+            "AmcConfigurations": [configuration_payload],
+            "apiOperation": "CREATE"
+        }
+        
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)